backend/helpers/crawlerHelper.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
```

Comment:
Rename class to `crawlersHelpers` as far as here potentially will be few helpers and we have few crawlers.

Reply:
Moved to `utils`